plutus-core/cost-model/budgeting-bench/Bench.hs

```diff
@@ -124,10 +141,30 @@ benchTwoInt builtinName =
 -- Creates the .csv file consumed by create-cost-model. The data in said csv is
 -- time taken for all the builtin operations, as measured by criterion.
 -- See also Note [Creation of the Cost Model]
+--
+-- TODO: Some care is required here regarding the current working directory.  If
+-- you run this benchmark via `cabal bench` or `stack bench` (but not `cabal
+-- run`) then the current directory will be `plutus-core`.  If you use nix it'll
+-- be the current shell directory, so you'll need to run it from `plutus-core`
+-- (NOT `plutus`, where `default.nix` is).  See SCP-2005.
 main :: IO ()
 main = do
-    createDirectoryIfMissing True "budgeting-bench/csvs/"
-    defaultMainWith (defaultConfig { C.csvFile = Just $ "budgeting-bench/csvs/benching.csv" }) $ (benchTwoInt <$> twoIntNames) <> (benchTwoByteStrings <$> [Concatenate]) <> (benchBytestringOperations <$> [DropByteString, TakeByteString]) <> (benchHashOperations <$> [SHA2, SHA3]) <> (benchSameTwoByteStrings <$> [EqByteString, LtByteString, GtByteString]) <> [benchVerifySignature] <> benchComparison
-    pure ()
-    where
-        twoIntNames = [AddInteger, SubtractInteger, MultiplyInteger, DivideInteger, QuotientInteger, RemainderInteger, ModInteger, LessThanInteger, LessThanEqInteger, GreaterThanInteger, GreaterThanEqInteger, EqInteger]
+  let dataDir = "cost-model" </> "data" :: FilePath
+      csvFile = dataDir </> "benching4.csv"
```

Comment:
benching4?

Reply:
Oooh, well-spotted.  That's left over from when I was trying to discover where the different tools were writing files.

Reply:
... and we don't need the type annotation on the preceding line either.